examples/reinforcement_learning/actor_critic_cartpole.py

```diff
@@ -0,0 +1,119 @@
+"""
+Title: Actor Critic Method
+Author: [Apoorv Nandan](https://twitter.com/NandanApoorv)
+Date created: 2020/05/13
+Last modified: 2020/05/13
+Description: Implement Actor Critic Method in CartPole environment
+"""
+"""
+## Setup
+"""
+
+import gym
+import numpy as np
+import tensorflow as tf
+from tensorflow import keras
+from tensorflow import math
+from tensorflow.keras import losses
+from tensorflow.keras import layers
+from tensorflow.keras import optimizers
+
+seed = 543
+gamma = 0.99
+env = gym.make("CartPole-v0")
+env.seed(seed)
+eps = np.finfo(np.float32).eps.item()
+
+"""
+## Implement Actor Critic network
```

Comment:
Please add an intro section explaining what this script does, why it matters, and provide a link to the reference paper(s).